index.html

```diff
@@ -228,7 +228,7 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 		</div>
 		<div class="row">
 			<div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 text-center">
-				<a class="mb0 btn btn-lg btn-filled" href="https://gci16.fossasia.org/#stprojects">Student Projects'16</a>
+				<a class="mb0 btn btn-lg btn-filled need" href="https://gci16.fossasia.org/#stprojects">Student Projects'16</a>
```

Comment:
please use something relevant class name instead of `need`

Reply:
😄